openwisp_monitoring/device/admin.py

```diff
@@ -8,17 +8,19 @@
 from swapper import load_model
 
 from openwisp_controller.config.admin import DeviceAdmin as BaseDeviceAdmin
-from openwisp_controller.config.models import Device
 
 from ..monitoring.admin import MetricAdmin
 from . import settings as app_settings
 
 DeviceData = load_model('device_monitoring', 'DeviceData')
 DeviceMonitoring = load_model('device_monitoring', 'DeviceMonitoring')
 Chart = load_model('monitoring', 'Chart')
+Device = load_model('config', 'Device')
 
 
 class DeviceAdmin(BaseDeviceAdmin):
+    change_form_template = 'admin/config/device/change_form.html'
```

Comment:
Why has it changed?

Reply:
I also would like to know this

Reply:
Without this change, `change_form_template` is fetched from [admin/config/change_form.html](https://github.com/openwisp/openwisp-controller/blob/master/openwisp_controller/config/templates/admin/config/change_form.html) from openwisp-controller, Previously, this wasn't required because path to this `change_form` was something like `admin/django_netjsonconfig/.../...`.

Reply:
I think I understand